.travis.yml

```diff
@@ -1,6 +1,6 @@
 # This Travis job script has been generated by a script via
 #
-#   runghc make_travis_yml_2.hs '-o' '.travis.yml' '--irc-channel=irc.freenode.org#haskell-lens' '--no-no-tests-no-bench' '--no-installed' 'cabal.project'
+#   runghc make_travis_yml_2.hs '-o' '.travis.yml' '--ghc-head' '--cabal-install-version=2.2' '--jobs=2' '--irc-channel=irc.freenode.org#haskell-lens' '--no-no-tests-no-bench' '--no-install-dependencies' '--config=cabal.haskell-ci' 'cabal.project'
```

Comment:
What is the purpose of the `--cabal-install-version=2.2` flag? AFAICT, it's not even doing what it advertises, since the matrix entries are all using `cabal-install-head` anyways.

(Personally, I prefer to use the earliest version of `cabal-install` that supports each version. But I can do this later if need be.)